rootfs-builder/README.md

```diff
@@ -0,0 +1,62 @@
+# Building a rootfs for Kata Containers Guest OS #
+
+The Kata Containers guest OS is created using `rootfs.sh`.
+
+## Supported base OS ##
+
+The `rootfs.sh` script support different based OS `rootfs`. To build
```

Comment:
I'm still struggling with this sentence a little. How about we change it to something like:

> The `rootfs.sh` script builds a rootfs based on a particular Linux* distribution. To build a `rootfs`for your chosen distribution, run:

Also, I wonder if we should add a default